src/test/ui/rfc-2565-param-attrs/param-attrs-pass.rs

```diff
@@ -0,0 +1,21 @@
+// compile-pass
+
+#![feature(param_attrs)]
+
+extern "C" {
+    fn ffi(#[cfg(bar)] a: u8, #[cfg(bar)] ...);
+}
+
+trait Trait2015 { fn bar(#[cfg(bar)] u8, #[cfg(bar)] u8); }
```

Comment:
I'd like to see a test file where this is not accepted in 2018.

Reply:
It isn't accepted because of the anonymous parameter, not the attribute itself (`anonymous parameters are removed in the 2018 edition (see RFC 1685)`).
Should a test be added anyway?

Reply:
>  Should a test be added anyway?

Yep. Redundancy + Tests = ❤️.